chill-akka/src/main/scala/com/twitter/chill/akka/ActorRefSerializer.scala

```diff
@@ -45,8 +41,8 @@ class ActorRefSerializer(system: ExtendedActorSystem) extends Serializer[ActorRe
   }
 
   override def read(kryo: Kryo, input: Input, typ: Class[ActorRef]): ActorRef = {
-    val path = input.readString()
-    system.actorFor(path)
+    val path = ActorPath.fromString(input.readString())
```

Comment:
Oh! Nice, I didn't realize that I DID merge these changes and that they're backwards compatible. Nice.